### ISSUE TITLE
Bad error message when no downloaders available

### ISSUE THREAD

**Issue by Tieske** [luarocks/luarocks#1221]
```
luarocks install --tree=system kong-jwt-claims-headers  --only-server=http://localhost:8080
[packer exec] luarocks install --tree=system kong-jwt-claims-headers  --only-server=http://localhost:8080

Error: LuaRocks 3.3.1 bug (please report at https://github.com/luarocks/luarocks/issues).
Arch.: linux-x86_64
/usr/local/share/lua/5.1/luarocks/fs/lua.lua:910: attempt to concatenate local 'downloader' (a nil value)
stack traceback:
	/usr/local/share/lua/5.1/luarocks/fetch.lua:69: in function 'fetch_url'
	/usr/local/share/lua/5.1/luarocks/fetch.lua:33: in function 'fetch_caching'
	/usr/local/share/lua/5.1/luarocks/manif.lua:108: in function 'load_manifest'
	/usr/local/share/lua/5.1/luarocks/search.lua:111: in function 'remote_manifest_search'
	/usr/local/share/lua/5.1/luarocks/search.lua:155: in function 'search_repos'
	/usr/local/share/lua/5.1/luarocks/search.lua:241: in function 'find_suitable_rock'
	/usr/local/share/lua/5.1/luarocks/search.lua:263: in function 'find_rock_checking_lua_versions'
	/usr/local/share/lua/5.1/luarocks/cmd/install.lua:239: in function </usr/local/share/lua/5.1/luarocks/cmd/install.lua:215>
	[C]: in function 'xpcall'
	/usr/local/share/lua/5.1/luarocks/cmd.lua:620: in function 'run_command'
	/usr/local/bin/luarocks:38: in main chunk
	[C]: at 0x55cfd1f91310
failed installing rock: 'luarocks install --tree=system kong-jwt-claims-headers  --only-server=http://localhost:8080 ' failed
ERROR: Service 'kong' failed to build: The command '/bin/sh -c /usr/local/openresty/luajit/bin/luajit /packer.lua -- "$INJECTED_PLUGINS"' returned a non-zero code: 1

```

This happens when no downloaders at all are available. Installing `wget` fixed the error, but the message generated by LuaRocks should probably be better.

**Comment by Tieske** [comment]
original issue: https://github.com/Kong/docker-kong/issues/374

**Comment by hishamhm** [comment]
@Tieske this is already fixed in `master` ( 568624c42d3ce2e0781e644e881f4b696424829f ), I need to get a new release out soon.
